spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe PasswordResetsController, :type => :controller do 
+
+  describe "GET show" do
+    it "renders show template if the token is valid" do
+      alice = Fabricate(:user)
+      alice.update_columns(token: "12345")
```

Comment:
Why are you updating the user instance? You should create it with the token already `alice = Fabricate(:user, token: "12345")`